spec/rubocop/formatter/offense_count_formatter_spec.rb

```diff
@@ -20,14 +20,14 @@ module Formatter
         context 'when no offenses are detected' do
           let(:offenses) { [] }
           it 'does not add to offense_counts' do
-            expect { finish }.not_to change { formatter.offense_counts }
+            expect { finish }.not_to(change { formatter.offense_counts })
```

Comment:
We can probably disable this cop completely for the `spec` folder in RuboCop and users might opt to do the same.